CHANGELOG.md

```diff
@@ -35,6 +35,9 @@
 
 ### Enhancements
 
+- Add each Loki instance as datasources in Grafana dashboards
+  (PR[#3681](https://github.com/scality/metalk8s/pull/3681))
+
```

Comment:
You will need to rebase since this should be in the 2.11.1 section (yet another issue with changelog as a simple file)